src/Runner.Worker/ActionManager.cs

```diff
@@ -682,12 +682,16 @@ private async Task BuildActionContainerAsync(IExecutionContext executionContext,
                         // * Repo or tag doesn't exist, or isn't public
                         // * Policy validation failed
                         if (ex is WebApi.UnresolvableActionDownloadInfoException)
-                        {
+                        {   // Log the error and fail the JobExtension Initialization.
+                            Trace.Error($"Caught exception from JobExtenion Initialization: {ex}");
+                            executionContext.InfrastructureError(ex.Message);
```

Comment:
I'm not convinced this should be considered infrastructure error. If the repo doesn't or they try to use a repo they don't have access to, should that be user error instead?